src/C++/DataDictionary.h

```diff
@@ -57,9 +57,6 @@ class DataDictionary
 
   struct MessageFieldsOrderHolder
   {
-    MessageFieldsOrderHolder()
-    {}
-
```

Comment:
Default constructor not needed for declaration when not doing anything additional.